_data/people/dynamic-entropy.yml

```diff
@@ -0,0 +1,11 @@
+active: true
+focus-area:
+  - XrootD
+  - FTS
```

Comment:
```
Check people files.......................................................Failed
- hook id: check-jsonschema
- exit code: 1

ok -- validation done
Schema validation errors were encountered.
  _data/people/dynamic-entropy.yml::$.focus-area[0]: 'XrootD' is not one of ['as', 'ia', 'ssc', 'doma', 'ssl', 'osglhc', 'core']
  _data/people/dynamic-entropy.yml::$.focus-area[1]: 'FTS' is not one of ['as', 'ia', 'ssc', 'doma', 'ssl', 'osglhc', 'core']
```

`focusa-area` is supposed to map to institute focus areas.

Reply:
Thank you @matthewfeickert 
I have addressed the changes requested in the validation check. 

Cheers